.github/workflows/release.yml

```
@@ -17,34 +17,69 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |-
           echo 'GITHUB_TOKEN=${{secrets.GITHUB_TOKEN}}' > .release-env
-      - name: draft release
+      - name: build and upload goreleaser binaries
         run: make release
+
+  rocksdb-x86_64:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    needs: ["goreleaser"]
+    runs-on: ${{ matrix.os }}
+    environment: release
+    steps:
+      - uses: cachix/install-nix-action@v16
+      - uses: cachix/cachix-action@v10
+        with:
+          name: cronos
+          signingKey: "${{ secrets.CACHIX_SIGNING_KEY }}"
+      - name: build linux/mac x86_64 rocksdb binaries
+        run: |
+          PLATFORM="$(uname -s)_x86_64"
+          FLAKE="github:${GITHUB_REPOSITORY}/${GITHUB_REF_NAME}"
+          nix build -L ${FLAKE}#cronosd-tarball
+          cp result cronos_${GITHUB_REF_NAME}-rocksdb_${PLATFORM}.tar.gz
```

```suggestion
          cp result cronos_${GITHUB_REF_NAME:1}-rocksdb_${PLATFORM}.tar.gz
```
remove the first character `v` to align with current release naming